src/app/mail/MailRecipients.ts

```diff
@@ -44,61 +44,92 @@ export function getRecipients(
 	includeEditors : boolean = false
 ) : Promise <Array <string>>
 {
+
+	// override emails for development
 	if (override && !app.isProduction) {
-		return Promise.resolve([ override ]);
+		return Promise.resolve([override]);
 	}
 
-	// If feedback email override to dedicated addresses is set for website
-	// then send messages there only instead of article authors
-	if (website.feedbackEmailOverride && website.feedbackEmailOverride.length) {
-		return Promise.resolve(website.feedbackEmailOverride);
+	const {
+		name,
+		chiefEditors,
+		overrideSettings: {
+			settings = {
+				feedback : false,
+			},
+			overrides = {
+				feedbackEmail: [],
+				fallbackFeedbackEmail: [],
+			},
+		} = {},
+	} = website;
+
+	const {feedbackEmail = [], fallbackFeedbackEmail = []} = overrides;
+
+	// if website is set to send all feedback's to dedicated addresses then override email addresses
+	if (settings.feedback && feedbackEmail.length) {
+		return Promise.resolve(overrides.feedbackEmail);
 	}
 
 	let recipients : Array <string> = filterForMailAddr(article.authors);
 
+	// if authors list is empty and we have fallback emails then add them to list
+	if (recipients.length <= 0 && fallbackFeedbackEmail.length) {
```

Comment:
Can the array have length less than 0? 
Just wondering

Reply:
This check was made by Philipp at start, so I really don't want to change it. Most things he wrote had some kind of genius idea, maybe it is one of them and has reason =)